blog/.vuepress/theme/components/Home.vue

```diff
@@ -24,7 +24,7 @@
         </div>
 
         <router-link
-          to="/posts"
+          to="/posts/"
```

Comment:
Just out of curiosity, why is this needed?

Reply:
The ultimate URL is /posts/ linking to a redirect might mess with our SEO score. And does certainly with our SEO Healthscore on ahrefs.com.

Reply:
Interesting, thanks Hannes!